grafana-plugin/src/containers/PluginConfigPage/PluginConfigPage.tsx

```diff
@@ -34,6 +30,11 @@ const cx = cn.bind(styles);
 
 interface Props extends PluginConfigPageProps<AppPluginMeta<OnCallAppSettings>> {}
 
+const constructSyncErrorMessage = (errMsg: string, url?: string): string =>
```

Comment:
I think it's better to  create file named `PluginConfigPage.helpers.ts` in the same folder and move those helper methods there